ui/src/pages/View.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {Component} from 'react';
+import {FormattedMessage, defineMessages, injectIntl} from 'react-intl';
+
+import LoadingIndicator from './../components/LoadingIndicator.js';
+import Utils from './../Utils.js';
+
+class View extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoaded: false,
+      error: null,
+      repo: null,
+      person: null
+    };
+  }
+
+  componentDidMount() {
+    const personId = Utils.getURLParam('id');
+    // TODO(nworden): consider if we could have a global cache of repo info to
+    // avoid calling for it on each page load
+    const apiURLs = [
+        '/' + this.props.match.params.repoId + '/d/repo',
+        '/' + this.props.match.params.repoId + '/d/person?id='
+            + encodeURIComponent(personId),
+        ];
+    Promise.all(apiURLs.map(url => fetch(url)))
+        .then(res => Promise.all(res.map(r => r.json())))
+        .then(
+          ([repo, person]) => {
+            this.setState({
+              isLoaded: true,
+              repo: repo,
```

Comment:
Optional: You can simplify `repo: repo` to `repo`.

Reply:
Oh, interesting. I think it's more readable this way though, so I'm going to leave it as it is.

Reply:
Yeah it is up to you, but I believe it's a commonly used feature of ES6, especially when all keys matches variable names like `{error: error}` to `{error}`.